projects/batfish/src/test/resources/org/batfish/grammar/cisco/testrigs/ibgp-reject-routerid-match/configs/r4

```diff
@@ -0,0 +1,35 @@
+! boot system flash:EOS.swi
+hostname r4
+!
+interface Ethernet1
+   no switchport
+   ip address 10.14.0.4/24
+!
+interface Ethernet2
+   no switchport
+   ip address 10.24.0.4/24
+!
+interface Ethernet3
+   no switchport
+   ip address 10.34.0.4/24
+!
+interface Loopback0
+   ip address 4.0.0.1/32
+!
+ip routing
+!
+router bgp 1
+   router-id 4.4.4.4
+   neighbor 10.14.0.1 remote-as 1
```

Comment:
Please use loopbacks to establish iBGP sessions. Add static routes to those loopbacks wherever they are referred to. The next hop should be the physical interface address of the neighbor.